assets/source/catalog-sync/sections/SyncState.test.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import SyncState from './SyncState';
+
+describe( 'SyncState component', () => {
+	test( 'should render header and footer correctly', () => {
+		const { getByText, getByRole } = render( <SyncState /> );
+
+		expect( getByRole( 'heading' ) ).toHaveTextContent( 'Overview' );
+
+		expect(
+			getByText( 'Set up and manage ads to increase your reach with' )
+		).toBeTruthy();
+
+		expect( getByRole( 'link' ) ).toHaveTextContent(
+			'Pinterest ads manager'
```

Comment:
💅

I would add `Pinterest ads manager(opens in a new tab)` to test as well if the link is ExternalLink.

Reply:
Spot on! Fixed in 424f5ec, thanks.